.gitignore

```diff
@@ -30,4 +30,9 @@ common/dist/scripts/*
 common/dist/sprites/habitrpg-shared.css
 .elasticbeanstalk/config.yml
 
+
 test/spec/mocks/translations.js
+
+#vim
+.tern-project
```

Comment:
How do I remove it? I'm really new to this.  If I do a rebase --onto and then push origin --force will that work or will I mess everything up?